gravitee-node-cluster/gravitee-node-cluster-plugin-hazelcast/src/main/java/io/gravitee/node/plugin/cluster/hazelcast/spring/HazelcastClusterConfiguration.java

```diff
@@ -13,80 +13,41 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.gravitee.node.cluster.spring;
+package io.gravitee.node.plugin.cluster.hazelcast.spring;
 
 import com.hazelcast.config.Config;
 import com.hazelcast.config.FileSystemXmlConfig;
 import com.hazelcast.config.FileSystemYamlConfig;
 import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.spi.properties.ClusterProperty;
-import io.gravitee.node.api.cache.CacheManager;
-import io.gravitee.node.api.cluster.ClusterManager;
-import io.gravitee.node.api.message.MessageProducer;
-import io.gravitee.node.cache.hazelcast.HazelcastCacheManager;
-import io.gravitee.node.cluster.hazelcast.HazelcastClusterManager;
-import io.gravitee.node.cluster.hazelcast.HazelcastMessageProducer;
 import java.io.FileNotFoundException;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.*;
-import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
 /**
- * @author Kamiel Ahmadpour (kamiel.ahmadpour at graviteesource.com)
+ * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
 @Configuration
-@Conditional(HazelcastClusterConfiguration.ClusterModeEnabled.class)
 public class HazelcastClusterConfiguration {
 
-    @Value("${cluster.hazelcast.config.path:${gravitee.home}/config/hazelcast.xml}")
+    @Value("${cluster.hazelcast.config.path:${gravitee.home}/config/hazelcast-cluster.xml}")
     private String hazelcastConfigFilePath;
 
     @Bean
-    public HazelcastInstance distributedHazelcastInstance() throws Exception {
+    public HazelcastInstance clusterHazelcastInstance() throws FileNotFoundException {
         // Force Hazelcast to use SLF4J before loading any HZ classes
         System.setProperty(ClusterProperty.LOGGING_TYPE.getName(), "slf4j");
         System.setProperty(ClusterProperty.SHUTDOWNHOOK_ENABLED.getName(), "false");
 
         Config config = fromFilePath(hazelcastConfigFilePath);
-
-        // Force the classloader used by Hazelcast to the container's classloader.
-        config.setClassLoader(ClusterConfiguration.class.getClassLoader());
-
```

Comment:
It is because it doesn't help anymore. This setting was used to be able to serializable cache object, here the hazlecast instance is only used for clustering. Also testing it with monitoring and HealthCheck didn't cause any issue regarding serialization.

Reply:
Ok let's remove it so 👍